packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -4,12 +4,16 @@ import {
   iconClass,
   selectClass,
   selectContainerClass,
+  selectVariants,
 } from './Select.css';
 
 import { SystemIcon } from '@components/Icon';
 import classNames from 'classnames';
 import React, { FC, forwardRef } from 'react';
 
+export type SelectVariants = 'form' | 'flat';
+export const variants: SelectVariants[] = ['form', 'flat'];
+
```

Comment:
I think this can be removed

Reply:
I did that so there's one source of truth and I can use it with the `variants[0]`, so we only have to change it in one location if we ever do.